src/cases/directives/show-allocate-link.directive.ts

```diff
@@ -0,0 +1,35 @@
+import { Directive, ElementRef, Input, OnInit } from '@angular/core';
+import { CaseRole, RoleType } from 'api/workAllocation2/interfaces/caseRole';
+
+@Directive({
+  selector: '[exuiShowAllocateLink]'
+})
+export class ShowAllocateLinkDirective implements OnInit {
+  private static CASE_MANAGERS_LIMIT = 1;
+  @Input() public showAllocateRoleLink: boolean = false;
+  @Input() public roles: CaseRole[];
+  @Input() public userType: string;
+
+  constructor(private element: ElementRef<HTMLElement>) {
+  }
+
+  private static canDisplayLink(roles: CaseRole[], userType: string, showAllocateRoleLink: boolean): boolean {
+    const show = false;
+    if (!showAllocateRoleLink || userType === 'legalOps' && ShowAllocateLinkDirective.hasExceededNumberCaseManagerRoles(roles)) {
```

Comment:
'legalOps' should be a constant/enum along with 'judiciary' so if we need to change any mention of it, looking through code is not necessary